.github/label-change.yml

```diff
@@ -31,3 +31,28 @@ ci:
   - ".github/**"
   - "dockers/**"
   - ".pre-commit-config.yml"
+
+docs:
+  - ".github/workflows/docs-build.yml"
+  - "docs/**"
+  - "requirements/docs.txt"
+  - "requirements/*/docs.txt"
```

Comment:
It's more appropiate that a PR that touches requirements is considered  a "dependencies PR". For instance, a dependabot PR is not a "docs PR"

```suggestion
```